functions/go/set-setter/main.go

```diff
@@ -0,0 +1,160 @@
+// This file will be processed and embedded to pluginator.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/set-annotation/setters"
+	"sigs.k8s.io/kustomize/api/resmap"
+	"sigs.k8s.io/kustomize/api/resource"
+	"sigs.k8s.io/kustomize/api/wrappy"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework"
+	kyaml "sigs.k8s.io/kustomize/kyaml/yaml"
+	"sigs.k8s.io/yaml"
+)
+
+type setSetterSpecs struct {
+	Setters []setSetterSpec `json:"setters,omitempty" yaml:"setters,omitempty"`
+}
+
+type setSetterSpec struct {
+	SetterName  string      `json:"setter_name,omitempty" yaml:"setter_name,omitempty"`
```

Comment:
I just found that we have some inconsistency about json/yaml tag naming in our existing code base. For example, we have `fieldSpecs` and `annotation_name`. `camelCase` is the standard in k8s world, so we should use it in our repo. @Shell32-Natsu 

Here we should use `setterName` and `setterValue` here.

Reply:
Created PR https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/170

Reply:
Updated this PR same as #170